### PR TITLE
PLF-7169 : Upgrade eXo Platform to ISPN last version (8)

### PR DESCRIPTION
**Pull request by aboughzela**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -111,6 +111,8 @@
     <version.plugin.japex>1.2.3</version.plugin.japex>
     <version.plugin.jibx>1.3.1</version.plugin.jibx>
 
+    <org.infinispan.version>8.2.6.Final</org.infinispan.version>
```
Comment (thomasdelhomenie) [inline pom.xml]: Same problem than with Groovy upgrade, it should be moved to gatein dependencies management pom.

File: pom.xml [hunk 2]
```diff
@@ -132,7 +134,7 @@
     <module>wsrp-integration</module>
     <module>mobile-integration</module>
     <module>cdi</module>
-    <module>packaging</module>
+<!--    <module>packaging</module> fixme resolve dependency-->
```
Comment (thomasdelhomenie) [inline pom.xml]: To be resolved.

Comment (aboughzela) [inline pom.xml]: This module should be removed after re-base gatein-portal ( packaging is removed from 5.x after upgrade wildfly and tomact 8 https://github.com/exodev/gatein-portal) 
I have disable this module due to conflict related to wsrp dependency.

File: pom.xml (hunk 1 above)
Comment (aboughzela) [inline pom.xml]: groovy version is declared on https://github.com/exodev/gatein-portal/blob/feature/upgrade-groovy/pom.xml#L72
gatein dependencies management pom (gatein-parent) is not forked from gatein to exo project

Comment (thomasdelhomenie) [inline pom.xml]: Yes, exactly, and we agreed with @mgreau to fork gatein-parent to update it in gatein-parent.

File: pom.xml (hunk 2 above)
Comment (thomasdelhomenie) [inline pom.xml]: JBoss and Tomcat have been upgraded, can you rebase now please ?

**Pull request merged**